### DISCUSSION TITLE
Interested to work on a project 🙌

### DISCUSSION THREAD

**Discussion by alvone01** [SeaQL/summer-of-code#3] [General]
Hi, I am Alvin, a final-year computer science student interested to work on a SeaQL project (preferably the first or second idea).
I have experience working in various companies in areas surrounding software development (web, mobile, libraries, blockchain) and am interested in distributed systems.

I am currently in the process of learning Rust and have previous experience using GraphQL with Typescript. I am excited to contribute to an OSS project and hope we can discuss soon!

**Comment by tyt2y3** [comment]
Welcome. Please follow https://github.com/SeaQL/summer-of-code/blob/main/2022/CONTRIBUTING.md

**Comment by billy1624** [comment]
Hey @alvone01, GSoC contributor application now opens. Please submit your proposal early so we can review and provide feedbacks :)

**Reply by alvone01** [reply]
Hi, thanks Billy. Since I mentioned I won't be able to complete the whole program period (only until August), I was potentially thinking of just contributing outside the scope of GSoC.

**Reply by billy1624** [reply]
Sure! Any form of contributions are welcomed
Thanks in advanced :P
